core/src/main/scala/chisel3/Mem.scala

```diff
@@ -484,7 +484,7 @@ sealed class SyncReadMem[T <: Data] private[chisel3] (
   )(
     implicit sourceInfo: SourceInfo
   ): T =
-    _readWrite_impl(idx, data, en, isWrite, clock, true)
+    _readWrite_impl(idx, data, en, isWrite, clock, false)
```

Comment:
I'm confused by this change, is this not a change in behavior for regular SyncReadMems? What does this have to do with the rest of this PR?

Reply:
I forgot to make this its own PR, but the SyncReadMem erroneously raises implicit clock warnings when an explicit clock is being used, and this change fixes that. I'm going to revert it since the warnings don't affect the SRAM itself